src/sync/treiber_stack.rs

```diff
@@ -77,6 +77,12 @@ impl<T> TreiberStack<T> {
     }
 }
 
+impl<T> Default for TreiberStack<T> {
+    fn default() -> Self {
+        TreiberStack { head: Atomic::null() }
```

Comment:
Hmm, interesting. Well, pretty much every `impl Default for ...` in rust-lang/rust relies on `Self::new()`. But I don't care too much either way. :)